cmd/openshift-install/create.go

```diff
@@ -289,49 +289,36 @@ func destroyBootstrap(ctx context.Context, config *rest.Config, directory string
 	events := client.CoreV1().Events("kube-system")
 
 	eventTimeout := 30 * time.Minute
-	logrus.Infof("Waiting up to %v for the bootstrap-complete event...", eventTimeout)
-	eventContext, cancel := context.WithTimeout(ctx, eventTimeout)
-	defer cancel()
-	_, err = Until(
-		eventContext,
-		"",
-		func(sinceResourceVersion string) (watch.Interface, error) {
-			for {
-				watcher, err := events.Watch(metav1.ListOptions{
-					ResourceVersion: sinceResourceVersion,
-				})
-				if err == nil {
-					return watcher, nil
-				}
-				select {
-				case <-eventContext.Done():
-					return watcher, err
-				default:
-					logrus.Warningf("Failed to connect events watcher: %s", err)
-					time.Sleep(2 * time.Second)
-				}
-			}
-		},
-		func(watchEvent watch.Event) (bool, error) {
-			event, ok := watchEvent.Object.(*corev1.Event)
-			if !ok {
-				return false, nil
-			}
+	logrus.Infof("Waiting up to %v for the bootstrap-success event...", eventTimeout)
+	if err := wait.PollImmediate(time.Second, eventTimeout, func() (done bool, err error) {
+		if _, err := events.Get("bootstrap-success", metav1.GetOptions{}); err == nil {
+			return true, nil
+		}
+		logrus.Warningf("Failed to get bootstrap-success event: %s", err)
```

Comment:
This is too much logging to the user console as default as INFO. please suppress them.

Reply:
changed to Debugf